modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -230,21 +244,32 @@ class ConnectionAvgStats {
 
                 this._avgRTT.appendReport(batchReport, props);
 
-                // Report end to end RTT only for JVB
-                if (!this.isP2P) {
-                    const avgRemoteRTT = this._calculateAvgRemoteRTT();
-                    const avgLocalRTT = this._avgRTT.calculate();
+                if (this.isP2P) {
+                    // Report RTT diff only for P2P.
+                    const jvbEnd2EndRTT = this._conference.avgRtpStatsReporter
```

Comment:
I see what you mean, I finally opted for the less invasive (but dirtier) approach, but I can gladly change it, if that's what you'd rather see here. Would you like me to proceed with the change?